src/Features/Traits/Console.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Features\Traits;
+
+use Behat\Gherkin\Node\TableNode;
+use Illuminate\Contracts\Console\Kernel;
+use Illuminate\Contracts\Container\BindingResolutionException;
+use Illuminate\Support\Facades\Artisan;
+use InvalidArgumentException;
+use PHPUnit\Framework\Assert;
+
+trait Console
+{
+    use Application;
+
+    protected string $consoleOutput = "";
+
+    /**
+     * @Given I run artisan command :command
+     * @Given I run artisan command :command in console
+     * @throws BindingResolutionException
+     */
+    public function runArtisanCommand(string $command): void
+    {
+        $this->consoleOutput = "";
+        $this->getContainer()->make(Kernel::class)->call($command);
+        $this->consoleOutput = $this->getContainer()->make(Kernel::class)->output();
+    }
+
+    /**
+     * @Given I run artisan command :command with
+     * @throws BindingResolutionException
+     */
+    public function runCommandWithOptionsAndArguments(string $command, TableNode $table): void
+    {
+        $options = "";
+        $arguments = "";
+
+        foreach ($table->getRows() as $row) {
+            if ($row[0] === "option") {
+                $options .= " --" . $row[1];
+            } elseif ($row[0] === "argument") {
+                $arguments .= " " . $row[1];
+            } else {
+                throw new InvalidArgumentException("Invalid key: $row[0]. Allowed keys are 'option' and 'argument'.");
+            }
+        }
+
+        $this->runArtisanCommand("$command $options $arguments");
+    }
+
+    /**
+     * @Then I see :output in console
```

Comment:
I don't see any other occurences of `@Then I...` in the codebase. Please be consistent with phrasing. 

I don't mean that's a wrong approach. But it should be same in all methods, so please discuss how should be handle this.